app/models/hyrax/permission_template.rb

```diff
@@ -10,9 +10,13 @@ module Hyrax
   class PermissionTemplate < ActiveRecord::Base
     self.table_name = 'permission_templates'
 
-    has_many :access_grants, class_name: 'Hyrax::PermissionTemplateAccess'
+    has_many :access_grants, class_name: 'Hyrax::PermissionTemplateAccess', dependent: :destroy
     accepts_nested_attributes_for :access_grants, reject_if: :all_blank
 
+    has_many :workflows, class_name: 'Sipity::Workflow', dependent: :destroy
```

Comment:
Can we give this a better name? `:workflow_options, :potential_workflows, :selectable_workflows, or :workflow_choices`?